app/scripts/proactive/app.js

```diff
@@ -205,6 +205,16 @@ define(
                 that.xmlToImport = new XMLSerializer().serializeToString(response);
                 dom.open_catalog_workflow();
             });
+        },
+        openWorkflowFromScheduler : function(jobId) {
+            var that = this;
+            var sessionId = localStorage['pa.session'];
+
+            var url = '/scheduler/portal/downloadjobxml?jobId='+jobId+'&sessionId='+sessionId;
+            dom.getWorkflowFromScheduler(url, function (response) {
+                that.xmlToImport = new XMLSerializer().serializeToString(response);
```

Comment:
why using `that` instead of `this` ?
If I understand well xmlToImport contains the response String. Where is it used ?

Reply:
I used seamless mechanism as catalog. `xmlToImport` is used by the import function to open the loaded `xml`.

Reply:
@MaelAudren you can approve pls?